scripts/shared.sh

```diff
@@ -39,6 +39,22 @@ function get_tfs_executable() {
     rm ${zip_file} && rm -rf exec_dir
 }
 
+function get_device_type() {
```

Comment:
I think it would be less confusing to explicitly show that providing EI results in cpu being selected here.

Reply:
user has to provide eia/gpu/cpu arg to script so is this method even needed?